src/main/resources/messages.properties

```diff
@@ -10,6 +10,7 @@ invalid.password=Your password must be between 8 and 30 characters at must conta
 invalid.username.size=Your login must be between 2 and 50 characters, do not use space
 invalid.token=You must be authenticated
 token.not.start.with.bearer=JWT token is not starts with Bearer prefix
+token.not.equal.to.your.token=You do not have access to view the data of another user!
```

Comment:
You don't use that message anywhere

Reply:
I know it doesn't apply. I'm a little confused as to where to apply it now or where to remove it at all. Since I was given a choice above to either output an informative message or ERROR 403.

Reply:
You might use it as an exception message using MessageSource.